databases/week3/sql_week3.sql

```diff
@@ -0,0 +1,237 @@
+CREATE DATABASE meal_sharing;
+USE meal_sharing;
+
+CREATE TABLE meal(
+ id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+ title VARCHAR (255),
+ description TEXT,
+ location VARCHAR(255) NOT NULL,
+ event_day DATETIME NOT NULL,
+ max_reservations INT NOT NULL,
```

Comment:
Could be unsigned since you won't be storing negative integers

Reply:
Yes added unsigned